crates/pathfinder/src/cairo/ext_py.rs

```diff
@@ -235,6 +241,27 @@ enum SubprocessExitReason {
     Shutdown,
     Death,
     Cancellation,
+    // If you add more reasons, remember to modify `all_labels`
```

Comment:
This could be solved by introducing a macro, especially that there's no explicit test for these metrics.

Reply:
I think that we should at all cost avoid having to remember such things, especially that metrics is public api that we will have to support and if there's room for regression we can easily remove, we should do it.

Reply:
Are you planning to add a lot of cases, and not see the warning in review? The enum hasn't changed in many months.

Reply:
Not really, I'm not even sure it is ever going to be changed. I was rather thinking in general, ie. the more metrics we have, the more probable it is to introduce subtle bugs because all variants|elements are not registered and some metric will not be available until the first increment.

Reply:
Well, the whole `extpy` will be removed at some point. I don't think this deserves an additional proc-macro.